backend/src/tasks/test/censysCertificates.test.ts

```diff
@@ -506,54 +506,4 @@ describe('censys certificates', () => {
 
     await checkDomains(organization);
   });
```

Comment:
All of these are decent tests, why would you remove them? Even if you created a function for the other tests, you still want to make sure that censysCertificates behaves appropriately. 

Please put these back.

Reply:
Done.